pyproject.toml

```diff
@@ -15,8 +15,14 @@ packages = [
 exclude = ["tests", "tests.*"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "3.9.*"
 "discord.py" = "^1.7.2"
+Sphinx = "^4.3.1"
+tomli = "^1.2.2"
+GitPython = "^3.1.24"
+sphinx-autodoc-typehints = "^1.12.0"
+releases = "^1.6.3"
+furo = "^2021.11.23"
```

Comment:
Shouldn't these be `dev-dependencies`? I'm not sure how `poetry` ships these upstream, but if we package, we probably don't need to have to install all these dependencies to use the repository?

Reply:
We could do it either way to be honest. I'll just move them I guess.

Poetry has an upcoming unreleased feature which allows grouping dependency in custom groups (think dev, docs, prod, testing, etc), but unfortunately that's not out yet. Waiting for the day it makes it out of alpha (though we could use alpha if we wanted to 👀).

Reply:
2196b4c